bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/BlueZBluetoothDevice.java

```diff
@@ -52,6 +53,8 @@
 
     private final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool("bluetooth");
```

Comment:
Pool name should be bluez releated as there is also BlueGiga adapter. I will implement similar functionality to BlueGiga adapter.

Reply:
Actually, there should be no pool at all. This feature just needs a single job from time to time. Best would be to use the `scheduler` that is available inside the Thing handler class.

Reply:
This is actually how I just implement it to BlueGiga. BluetoothDevice just keep on track the time of last received scan event and bridge will do the cleanup.

Reply:
Well, I have a followup PR that reworks and simplifies the logic in the BlueZBridgeHandler. Would it be alright if I make the change there instead?